crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2151,33 +2144,7 @@ impl AuthorityPerEpochStore {
         Ok(())
     }
 
-    /// Notifies that a synced checkpoint of sequence number `checkpoint_seq` is available. The source of the notification
-    /// is the CheckpointExecutor. The consumer here is guaranteed to be notified in sequence order.
-    pub fn notify_synced_checkpoint(&self, checkpoint_seq: CheckpointSequenceNumber) {
-        let mut highest_synced_checkpoint = self.highest_synced_checkpoint.write();
-        *highest_synced_checkpoint = checkpoint_seq;
-        self.synced_checkpoint_notify_read
-            .notify(&checkpoint_seq, &());
-    }
-
     /// Get notified when a synced checkpoint of sequence number `>= checkpoint_seq` is available.
```

Comment:
did you also mean to remove this comment?

Reply:
Whoops!